### PR TITLE
KAFKA-3709: Create a project security page.

### PR DESCRIPTION
**Pull request by fpj**

### CODE REVIEW

File: includes/header.html [hunk 1]
```diff
@@ -50,6 +50,7 @@
 				<li><a href="/performance.html">performance</a></li>
 				<li><a href="https://cwiki.apache.org/confluence/display/KAFKA/Clients">clients</a></li>
 				<li><a href="https://cwiki.apache.org/confluence/display/KAFKA/Ecosystem">ecosystem</a></li>
+                                <li><a href="/project-security.html">security</a></li>
```
Comment (ijuma) [inline includes/header.html]: Minor indenting issue here.

**Comment by ijuma** [comment]
LGTM apart from one trivial comment. @junrao, can you please take a look as well?

**Comment by fpj** [comment]
@ijuma indentation fixed.

**Comment by junrao** [comment]
Thanks for the patch. LGTM. I initiated the request to create the security mailing list for Kafka. Will update the PR when it's created.

**Comment by junrao** [comment]
The mailing list has been created. We can invite all committers/contributors who are interested in security to join the list.

  security@kafka.apache.org
        Moderators: junrao@gmail.com,ismael@juma.me.uk,cshapi@gmail.com
        This list is private.

**Comment by ijuma** [comment]
Thanks Jun. I emailed active committers informing them about the creation of the mailing list. I sent an email to the list from a different account and verified that messages are going through.

I'll merge this PR tomorrow morning if there are no objections.

**Pull request closed without merging**